configurations/main-footer.development.config.json

```diff
@@ -1,180 +1,190 @@
-          "label": "Member States",
-          "url": "https://eosc-portal.eu/documents/member-states"
-        }
-      ]
-    },
-    {
-      "label": "For providers",
-      "url": "https://eosc-portal.eu/for-providers",
-      "navBtns": [
-        {
-          "label": "Providers Documentation",
-          "url": "https://eosc-portal.eu/providers-documentation"
-        }
-      ]
-    },
-    {
-      "label": "Subscribe",
-      "url": "https://eosc-portal.eu/subscribe",
-      "navBtns": [
-        {
-          "label": "EOSC Portal FAQs",
-          "url": "https://eosc-portal.eu/about/faqs"
-        },
-        {
-          "label": "Term of use - Search Service",
-          "url": "https://search.eosc-portal.eu/acceptable-use-policy"
-        }
-      ]
-    },
-    {
-      "label": "Using the portal",
-      "url": "https://eosc-portal.eu/using-the-portal"
-    }
+    [
+      {
+        "label": "About",
+        "url": "https://eosc-portal.eu/about-eosc-portal",
+        "navBtns": [
+          {
+            "label": "EOSC",
+            "url": "https://eosc-portal.eu/about/eosc"
+          },
+          {
+            "label": "EOSC Portal",
+            "url": "https://eosc-portal.eu/about-eosc-portal"
+          },
+          {
+            "label": "EOSC Doers",
+            "url": "https://eosc-portal.eu/about/eosc-doers"
+          },
+          {
+            "label": "EOSC Glossary",
+            "url": "https://eosc-portal.eu/glossary"
+          },
+          {
+            "label": "EOSC Projects",
+            "url": "https://eosc-portal.eu/about/eosc-projects"
+          },
+          {
+            "label": "EOSC Governance",
+            "url": "https://eosc-portal.eu/governance"
+          },
+          {
+            "label": "EOSC Portal FAQs",
+            "url": "https://eosc-portal.eu/about/faqs"
+          },
+          {
+            "label": "EOSC Association",
+            "url": "https://eosc.eu/"
+          },
+          {
+            "label": "EOSC Enhance",
+            "url": "https://eosc-portal.eu/enhance"
+          }
+        ]
+      }
+    ],
+    [
+      {
+        "label": "Use Cases",
+        "navBtns": [
+          {
+            "label": "Community Use Cases",
+            "url": "https://eosc-portal.eu/eosc-in-practice/use-cases"
+          },
+          {
+            "label": "Submit Use Case",
+            "url": "https://eosc-portal.eu/eosc-in-practice/open-call"
+          },
+          {
+            "label": "Industry Use Case",
+            "url": "https://eosc-portal.eu/industry-use-cases"
+          }
+        ]
+      },
+      {
+        "label": "For providers",
+        "url": "https://eosc-portal.eu/for-providers",
+        "navBtns": [
+          {
+            "label": "Providers Documentation",
+            "url": "https://eosc-portal.eu/providers-documentation"
+          },
+          {
+            "label": "Promote your services",
+            "url": "#"
```

Comment:
Is this URL correct? In other places, it is `"https://www.eosc-portal.eu/contribute-promotional-article-your-service"`